### DISCUSSION TITLE
add relational fields support to CALENDAR LAYOUT start/end date fields

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#6000] [Feature Requests]
like display template does, being able to select a field in a related collection

**Comment by rijkvanzanten** [comment]
This would only apply to many to one fields I imagine? 🤔 Do you have a specific use case in mind?

**Reply by wc-matteo** [reply]
I have a table A with a translation table A_T. Date fields are in A, the title is in A_T. Right now I can't have the calendar layout with the start/end date and the title mapped correctly. Cause the only way to have the dates mapped is to start with table A but that way the title in A_T is not shown (cause it's o2m), I can only select the field `translations` and I can't go into A_T from that. It seems even the `display` in the o2m relationship (for the `translations` field) is ignored. It just shows A_T pks:

![image](https://user-images.githubusercontent.com/10562635/120515322-7f382f80-c3ce-11eb-9244-da5a657ad44c.png)

the display works correctly in the table layout:

![image](https://user-images.githubusercontent.com/10562635/120517033-48fbaf80-c3d0-11eb-9776-e054fc5abd32.png)

**Reply by wc-matteo** [reply]
@rijkvanzanten @benhaynes 

btw is there a `display` or a feature in the works to display a single value for a o2m field?

Like instead of showing Translations `3 items`, showing directly 1 of the items with a display template (like m2o)

I saw something about virtual columns. Maybe that could be used to show only 1 item from a o2m field, using a custom filter

**Reply by rijkvanzanten** [reply]
Not actively in the works, no. Sounds like a useful one to have though!

> I saw something about virtual columns. Maybe that could be used to show only 1 item from a o2m field, using a custom filter

That would apply to this as well, yeah 🤔

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
